### PR TITLE
Adds ulauncher settings and theme

### PR DESCRIPTION
**Pull request by simon-bueler**
Signed-off-by: simon-bueler <simon.bueeler@icloud.com>

### CODE REVIEW

File: community/sway/etc/sway/definitions [hunk 1]
```diff
@@ -23,7 +23,7 @@ set $term_float_portrait $term float_portrait
 # Your preferred application launcher
 # Note: pass the final command to swaymsg so that the resulting window can be opened
 # on the original workspace that the command was run on.
-set $menu exec $term_float_portrait sway-launcher-desktop
+set $menu exec GDK_BACKEND=wayland ulauncher --no-window-shadow
```
Comment (boredland) [inline community/sway/etc/sway/definitions:26]: couldn't you set "GTK_BACKEND" in the .profile like the other similar configs?
https://github.com/Manjaro-Sway/desktop-settings/blob/9eae1e81164d24405a5aeb5da4451d7108df7beb/community/sway/etc/skel/.profile#L12

**Review by boredland — CHANGES_REQUESTED** [review]
please check if this works with `GDK_BACKEND` set in .profile too, since other applications might profit from that var.

File: community/sway/etc/sway/config.d/98-application-defaults [hunk 2]
```diff
@@ -15,6 +15,7 @@ for_window [title="Save File"] floating enable
 for_window [title="Firefox — Sharing Indicator"] floating enable
 for_window [title=".* is sharing your screen."] floating enable
 for_window [app_id="wdisplays"] floating enable
+for_window [app_id="ulauncher"] floating enable, border none
```
Comment (boredland) [inline community/sway/etc/sway/config.d/98-application-defaults:18]:
```suggestion
for_window [app_id="ulauncher"] floating enable, border none, sticky enable
```

File: community/sway/etc/sway/definitions (hunk 1 above)
Comment (simon-bueler) [reply community/sway/etc/sway/definitions]: Removed GDK_BACKEND=wayland  since it not seem to be necessary

**Pull request merged**